llvm/test/MC/AMDGPU/gfx12_asm_vop3cx_dpp8.s

```diff
@@ -1,5 +1,4 @@
-// RUN: llvm-mc -triple=amdgcn -mcpu=gfx1200 -mattr=+wavefrontsize32 -show-encoding %s | FileCheck --check-prefixes=GFX12 %s
-// RUN: llvm-mc -triple=amdgcn -mcpu=gfx1200 -mattr=+wavefrontsize64 -show-encoding %s | FileCheck --check-prefixes=GFX12 %s
```

Comment:
This is losing test coverage

Reply:
Would you please tell how the test coverage gets lost by the change? The WaveSizePredicate is not set when we define V_OPCX_* instructions. This means it is for sure to generate the same code for Wave64 and Wave32. And there is no reason to check both.

Reply:
And that should be verified as working correctly on both

Reply:
Okay. you are thinking every instruction should be verified on every possible wavefrontsize.
This patch thinks if we are 100% sure same instruction will be generated on both wave sizes, we only need to verify for one wave size.  Both are reasonable and will not lose anything.
I am going to abandon this patch if there is no other opinions. Thanks.

Reply:
Yes, we also need to test the different paths that produce the same output